lib/core/constants/assets.dart

```diff
@@ -29,6 +29,16 @@ class AppIcons {
   static const upwardArrow = '$baseIconPath/upward_arrow.svg';
   static const downwardArrow = '$baseIconPath/downward_arrow.svg';
 
+  // Category Icons (temporary mapping to existing icons)
+  static String get trendingIcon => '$baseIconPath/trend_up.svg';
+  static String get sportsIcon => '$baseIconPath/SoccerBall.svg';
+  static String get entertainmentIcon => '$baseIconPath/music.svg';
+  static String get politicsIcon => '$baseIconPath/law.svg';
+  static String get cryptoIcon => '$baseIconPath/coins.svg';
+  static String get stocksIcon => '$baseIconPath/chart.svg';
+  static String get esportsIcon => '$baseIconPath/sword.svg';
+  static String get gamesIcon => '$baseIconPath/gamepad.svg';
```

Comment:
```suggestion
  static const gamesIcon = '$baseIconPath/gamepad.svg';
```

and for all above too